eth/stagedsync/stage_txpool.go

```diff
@@ -189,8 +200,17 @@ func unwindTxPoolUpdate(from, to uint64, pool *core.TxPool, db *ethdb.ObjectData
 			return true, nil
 		}
 
+		bodyRlp := v
+		if debug.IsBlockCompressionEnabled() && len(v) > 0 {
```

Comment:
Can we extract this and similar places into a function? Also replace "%s" -> "%w"

Reply:
done